app/controllers/provider/admin/dashboards_controller.rb

```diff
@@ -29,16 +29,12 @@ def show
 
   private
 
-  def notification_presenter
+  def notifications_presenter
     ::Dashboard::NotificationsPresenter.new(current_user.notifications)
   end
 
-  def messages_presenter
-    ::Dashboard::MessagesPresenter.new(current_account.received_messages)
-  end
```

Comment:
Removed because it's unused AFAIK

Reply:
This will require all necessary cleanup, be it in this PR or another one.